src/pkg/repository/repository_unexported_test.go

```diff
@@ -25,22 +39,66 @@ func TestRepositoryModelSuite(t *testing.T) {
 	})
 }
 
-func (suite *RepositoryModelSuite) TestWriteGetModel() {
+func (suite *RepositoryModelIntgSuite) SetupSuite() {
 	ctx, flush := tester.NewContext()
 	defer flush()
 
 	var (
-		t        = suite.T()
-		s        = tester.NewPrefixedS3Storage(t)
-		kopiaRef = kopia.NewConn(s)
+		t   = suite.T()
+		s   = tester.NewPrefixedS3Storage(t)
+		k   = kopia.NewConn(s)
+		err error
 	)
 
-	require.NoError(t, kopiaRef.Initialize(ctx))
-	require.NoError(t, kopiaRef.Connect(ctx))
+	require.NotNil(t, k)
+	require.NoError(t, k.Initialize(ctx))
 
-	defer kopiaRef.Close(ctx)
+	suite.kopiaCloser = func(ctx context.Context) {
+		k.Close(ctx)
+	}
```

Comment:
it's safe to call close on the returned conn as soon as you have a handle to the wrapper or model store. It's ref counted internally so you shouldn't get errors if you close it when you exit this function

Reply:
hmm, this pattern appears in many places.  We might want to do a tech-debt sweep for it.